models/demos/t3000/llama2_70b/scripts/repack_weights.py

```diff
@@ -87,4 +92,4 @@ def repack(in_dir, out_dir, chunk_size, num_layers, hidden_size):
     parser.add_argument("num_layers", type=int, default=80, help="total number of layers")
     parser.add_argument("hidden_size", type=int, default=8192, help="hidden size of the model")
     args = parser.parse_args()
-    repack(args.in_dir, args.out_dir, args.chunk_size, args.num_layers, args.hidden_size)
+    repack(args.in_dir, args.out_dir, args.chunk_size)
```

Comment:
Could you rebase and undo this change to the repack call? Someone merged a PR yesterday which fixed the default args issue. If you can pick that up and add your params.json copying, that would be great! https://github.com/tenstorrent/tt-metal/pull/11297#event-13859102211

Reply:
Done, thanks!